texttospeech/snippets/synthesize_long_audio.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# All Rights Reserved.
+
+"""Google Cloud Text-To-Speech API sample application .
+
+Example usage:
+    python synthesize_long_audio.py --text "hello"
+"""
+
+import argparse
+
+
+# [START tts_synthesize_long_audio]
+def synthesize_long_audio(parent, text, output_gcs_uri):
+    """Synthesizes speech from the input string of text."""
+    from google.cloud import texttospeech
+    
+    client = texttospeech.TextToSpeechLongAudioSynthesizeClient()
+
+    input_text = texttospeech.SynthesisInput(text=text)
+
+    voice = texttospeech.VoiceSelectionParams(
```

Comment:
Consider making these constants, at the top of the sample. (Same comment applies to other options specified throughout sample.)